interoperability.md

```diff
@@ -36,6 +36,25 @@ Having established which reports they want their app to process, the app develop
 
 When new TCN coalition members launch their own apps, or launch new partnerships with public health authorities to authenticate positive test results, other TCN coalition members can decide for themselves whether to trust and use the new reports. If they choose to do so, they can add support by simply updating their own servers’ or apps’ configuration files.
 
+## Common TCN API
+
+By implementing the following API, other apps who implement the TCN protocol will be able to fetch report data that was uploaded to other coalition member servers. **Note that the API doesn't enforce server-to-server nor client-to-server.** Rather it is flexible so those architectural decisions can be left to the team implementing the TCN protocol in their app.
+
+Further documentation on the API can be found in the [API definition](https://github.com/Co-Epi/coepi-backend-aws/blob/master/api_definition/coepi_api_0.4.0.yml).
+> Note: this link needs to be updated to point to a reference implementation on the TCN repo when we get there.
+
+##### API Endpoints
+| Method | Endpoint | Description |
+| ------ | ------------ | ------------------------ |
+| GET | /tcnreport | Returns a list of **signed** reports ('rvk \|\| tck_{j1-1} \|\| le_u16(j1) \|\| le_u16(j2) \|\| memo', refer to the [TCN protocol](https://github.com/TCNCoalition/TCN/blob/main/README.md)) concatenated together for a given time interval number. |
+
+##### Query Parameters
+| Name | Description |
+| -------- | ---------------------- |
+| intervalNumber | **Required:** Positive integer that corresponds to a specific fixed time interval and can be calculated as (unix_time / time_interval_ms) |
+| date | **Optional:** Date in RFC3339 standard in UTC, without the time component. If not provided, default value of today's date is used |
+| intervalLengthMs | **Optional:** The interval length in milliseconds used by the client to calculate intervalNumber. The server will respond with a 401 if the interval length does not match the expected length. |
```

Comment:
If intervalNumber is provided, this is not optional

Reply:
Thank you @bhushanRamnani, I have updated the doc :)